api/data_explorer/controllers/facets_controller.py

```diff
@@ -69,7 +67,8 @@ def facets_get(filter=None, extraFacets=None):  # noqa: E501
     :type extraFacets: List[str]
     :rtype: FacetsResponse
     """
-    _process_extra_facets(extraFacets)
+    es = Elasticsearch(current_app.config['ELASTICSEARCH_URL'])
```

Comment:
Why did this change? I don't see `es` being used anywhere.

Reply:
Sorry was going to use elasticsearch.get_field_type here but the facet_info type ended up being sufficient. Reverted that part.